pynest/nest/lib/hl_api_simulation.py

```diff
@@ -84,8 +83,14 @@ def Run(t):
     `Prepare` must be called before `Run` to calibrate the system, and
     `Cleanup` must be called after `Run` to close files, cleanup handles, and
     so on. After `Cleanup`, `Prepare` can and must be called before more `Run`
-    calls. Any calls to `SetStatus` between `Prepare` and `Cleanup` have
-    undefined behaviour.
+    calls.
+
+    Be careful about modifying the network or neurons between `Prepare` and `Cleanup`
+    calls. In particular, do not call `Create`, `Connect`, or `SetKernelStatus`.
+    Calling `SetStatus` to change membrane potential `V_m` of neurons or synaptic
+    weights (but not delays!) will in most cases work as expected, while changing
+    membrane or synaptic times constants **will not work correctly**. If in doubt, assume
```

Comment:
I thought you wanted to remove the bold part (I don't mind though)

Reply:
Thanks for catching this, dangerous to have essentially the same comment in two places ;). Fixed now.